@stellar/typescript-wallet-sdk/test/server.test.ts

```diff
@@ -57,3 +57,15 @@ describe("SEP-10 helpers", () => {
     expect(isValid).toBeFalsy();
   });
 });
+
+describe("Server helpers", () => {
+  it("should parse a JSON AnchorTransaction", () => {
```

Comment:
Can you add a test for MGI incomplete transactions?

Reply:
do you have a json string by chance? I'm having trouble generating one

Reply:
I'll get one

Reply:
hey @Ifropc were you able to get one?